### ISSUE TITLE
GeoAI use-case map page

### ISSUE THREAD

**Issue by heamart** [Canadian-Geospatial-Platform/geo.ca#278]
new content to be added for the GeoAI page. It will be another card under the map and dataset gallery. The page will be the same type as the map index page but showing use-case scenarios.

Will update issue with content and web services for the map soon.

**Comment by heamart** [comment]
Content owner has made the config files for the map page, both english and french:
Just waiting on the text to add to the GeoAI landing page and to the map page. Tentative page title will be "Urban Development Use-Case"

[GeoAI Urban Development, Quebec City EN-V02.json](https://github.com/Canadian-Geospatial-Platform/geo.ca/files/13439285/GeoAI.Urban.Development.Quebec.City.EN-V02.json)

[GeoAI Urban Development, Quebec City FR-V02.json](https://github.com/Canadian-Geospatial-Platform/geo.ca/files/13439287/GeoAI.Urban.Development.Quebec.City.FR-V02.json)

**Comment by heamart** [comment]
english content ready. Once up on dev for review the content owner will get the translation

On the geoai landing page, a third card can be added under the map and dataset gallery:
 
![urban-development-thumbnail](https://github.com/Canadian-Geospatial-Platform/geo.ca/assets/107938467/4dbf2d7c-2440-4be8-b5bf-d107dbad5656)
**Urban Development Use Case**
Explore the results of a change detection analysis in Québec City.

and the text for the new map page. The header image won't be the one used, we're looking for a different one:
[GeoAI Urban Development Map_EN (1).docx](https://github.com/Canadian-Geospatial-Platform/geo.ca/files/13546164/GeoAI.Urban.Development.Map_EN.1.docx)

**Comment by heamart** [comment]
also, some text updates to the GeoAI landing page english version. updates are in track changes.
[GeoAI_EN_v2 (2).docx](https://github.com/Canadian-Geospatial-Platform/geo.ca/files/13547819/GeoAI_EN_v2.2.docx)

**Comment by jennywilsonsewell** [comment]
@heamart @Andrea-Merry @jaredkinger 
I've added the Urban Development Use Case card to the [GeoAI landing page](https://wordpress.dev.geo.ca/initiatives/geobase/geoai/) in the Map and Dataset Gallery section. I've also updated the text content throughout the page.

**Comment by heamart** [comment]
@jaredkinger 
new header image to use on use-case map page: https://unsplash.com/photos/a-scenic-view-of-a-city-surrounded-by-mountains-GyE8UNeVWT4

still waiting on french translation for text on the page

**Comment by heamart** [comment]
ok here is the basics of the french text and where it should go in brackets, there may be edits that come in next week during QA/QC

- 1st, for the geoai homepage. the thumbnail in the added box for map will be the same as the english page:
[card title] Cas d’utilisation du développement urbain
[card text] Explorez les résultats d’une analyse de détection des changements dans la ville de Québec.

-2nd, for the map page itself. The header image will be the same as the english page.
[page title] **Cas d’utilisation du développement urbain** 
[header text] Explorez les résultats d’une analyse visant à détecter les zones de développement urbain

[sub-title] Développement urbain de la ville de Québec
[paragraph] La carte ci-dessous affiche les résultats d’une analyse temporelle du développement urbain de la ville de Québec à l’aide de la série GéoIA.
Les entités, telles que les bâtiments et les routes, ont été extraites des images satellitaires de 2006 et de 2022. Ensuite, une analyse simple a été effectuée pour détecter les entités présentes en 2022 qui n’existaient pas en 2006. L’ajout de nouvelles entités urbaines, comme les bâtiments, est un bon indicateur du développement ou de l’étalement urbain.

**Comment by jaredkinger** [comment]
@heamart
I've updated the [French Use Map Page](https://wordpress.dev.geo.ca/fr/initiatives-fr/geobase/geoia/cas-dutilisation-du-developpement-urbain/) and the [GeoAI French page](https://wordpress.dev.geo.ca/fr/initiatives-fr/geobase/geoia/) up to date.

**Comment by heamart** [comment]
A few updates if there is still time

1. the french went weird on the GeoAI french page. should be:
Title: Cas d’utilisation du développement urbain
Text: Explorez les résultats d’une analyse de détection des changements pour la ville de Québec.

2. Update json configuration file for french map (basemaps were the english version)
[geoai urban development quebec city FR_V05.json](https://github.com/Canadian-Geospatial-Platform/geo.ca/files/13648714/geoai.urban.development.quebec.city.FR_V05.json)

3. GEO.ca homepage - the GéoIA link under "En vedette" needs the space removed between "Géo" and "IA"

**Comment by heamart** [comment]
minor text update to english and french if there is still time, if not it can wait to next release.
Last sentence in the paragraph ("the addition of new....) will change to:

English: The addition of features like buildings and roads is a good indicator of urban development or sprawl.
French: L'ajout d'entités comme les bâtiments et les routes est un bon indicateur du développement ou de l'étalement urbain.

**Comment by jaredkinger** [comment]
@heamart I've updated this page and pushed this over to staging.

EN: https://stage.geo.ca/initiatives/geobase/geoai/urban-development-use-case/
FR: https://stage.geo.ca/fr/initiatives-fr/geobase/geoia/cas-dutilisation-du-developpement-urbain/

I noticed that the dataset we are linking to in the "View Dataset" button is currently returning no results. Will that dataset be published in the upcoming release?

**Comment by heamart** [comment]
@jaredkinger no that was my mistake, I put the wrong links to the dataset

english: https://app.geo.ca/result/en/geoai---geobase-series?id=74738ff5-5367-5958-9aee-98fffdcd1876&lang=en
french: https://app.geo.ca/result/fr/g%C3%A9oia---s%C3%A9rie-g%C3%A9obase?id=74738ff5-5367-5958-9aee-98fffdcd1876&lang=fr

**Comment by Andrea-Merry** [comment]
@jaredkinger @heamart 
The last requested change from Heather is not on the English page. This is what is there.
Features, such as buildings and roads, were extracted from satellite images from 2006 and 2022. Then, a simple analysis was done to detect features present in 2022 that were not there in 2006. [Click here to join the meeting]

NOTE: There is a link to a teams meeting after the sentence. Please remove.
French page is ok

**Comment by heamart** [comment]
In addition to Andrea's comment, the last sentence is missing after "...not there in 2006." should be the following:

The addition of features like buildings and roads is a good indicator of urban development or sprawl.

**Comment by jaredkinger** [comment]
@Andrea-Merry @heamart 

Sorry about the Teams link in there. Sometimes the clipboard doesn't update when copying and pasting over telework. Not sure how I missed that error but thank you for catching it!

I've updated this on staging and it should have the correct copy now. 
https://stage.geo.ca/initiatives/geobase/geoai/urban-development-use-case/

**Comment by Andrea-Merry** [comment]
@jaredkinger @heamart 
Thank you. Checked and good to go.
